app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template">
   <head>
-    <title><%= yield(:title).present? ? yield(:title) : t('service_name') %></title>
+    <title><%= yield(:title).present? ? browser_title(yield(:title)) : t('service_name') %></title>
```

Comment:
This is close, but if there’s no `:title`, the browser title will be `[Service name]` not `[Service name] - GOV.UK`.

I wonder if we can use the helper like this here:

```suggestion
    <title><%= browser_title(yield(:title)) %></title>
```

And then in the helper, if a title is present, output:

`[title] - [Service name] - GOV.UK`

and if no title is given, output:

`[Service name] - GOV.UK`